Components/Badge/badge.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="badge.css">
+    <link rel="stylesheet" href="/css/style.css">
+    <script src="js/fontawesomepro.js"></script>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.0.0/css/all.css">
+
+    <title>Mango UI</title>
+</head>
+
+<body>
+    <nav>
+        <div class="nav-hero">
+
+            <a href="#"> Mango UI <i class="fa-solid fa-mango"></i> </a>
+        </div>
+
+        <ul class="nav-pills">
+            <li><a href="index.html">Home <i class="fa-solid fa-house-chimney"></i></a></li>
+            <li><a href="https://github.com/Shivanshsharma01/Mango-UI">Github <i class="fa-brands fa-github"></i></a>
+            </li>
+
+        </ul>
+    </nav>
+    <div class="container">
+
+
+        <div class="side-content">
+            <div class="sidebar">
```

Comment:
you can use semantic HTML's aside tag here

Reply:
This div is for whole main content, maybe aside tag will not be appropriate for this